plugins/plogs.yaml

```
@@ -0,0 +1,73 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: plogs
+spec:
+  version: v0.1.1
+  homepage: https://github.com/kha7iq/plogs
+  shortDescription: Retrieve and stream colorized logs from Kubernetes pods
```

```suggestion
  shortDescription: Retrieve and stream colorized logs from Pods
```

- it's clear we're in Kubernetes context as a kubectl plugin :)
- capitalized API names is the preferred writing style for docs.

you can fix pods → Pods below as well.

---

Fixes applied , and yes Kubernetes is implied :)